org.freedesktop.Sdk.Extension.ziglang.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "org.freedesktop.Sdk.Extension.ziglang",
+    "default-branch": "19.08",
+    "branch": "19.08",
+    "runtime": "org.freedesktop.Sdk",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "build-extension": true,
+    "separate-locales": false,
+    "appstream-compose": false,
+    "build-options": {
+        "prefix": "/usr/lib/sdk/ziglang"
+    },
+    "modules": [
+        {
+            "name": "zig",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /usr/lib/sdk/ziglang",
+                "cp -r ./* /usr/lib/sdk/ziglang",
+                "chmod a+x /usr/lib/sdk/ziglang/zig"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://ziglang.org/download/0.6.0/zig-linux-x86_64-0.6.0.tar.xz",
+                    "sha256": "08fd3c757963630645441c2772362e9c2294020c44f14fce1b89f45de0dc1253",
+                    "only-arches": [
+                        "x86_64"
+                    ]
+                },
+                {
+                    "type": "archive",
+                    "url": "https://ziglang.org/download/0.6.0/zig-linux-i386-0.6.0.tar.xz",
+                    "sha256": "a97a2f9ae21575743cdd763c1917d49400d83fc562ef64582b18bade43eb24ce",
+                    "only-arches": [
+                        "i386"
+                    ]
+                },
```

Comment:
The recent runtimes don't support i386 anymore. So this can be removed
```suggestion
```

Reply:
Sure, but this manifest is for Freedesktop SDK 19.08, so if 19.08 supports i386, I would like this package to support it as well.

Reply:
I'll apply your change later today I guess.

Reply:
19.08 doesn't already, hence my change suggestion